cmd/flags.go

```diff
@@ -172,6 +172,11 @@ func dataCacheFlags() []cli.Flag {
 			Value: "300M",
 			Usage: "total read/write buffering in MiB",
 		},
+		&cli.StringFlag{
+			Name:  "max-readahead",
+			Value: "32M",
```

Comment:
A 32MiB default setting isn't necessary here.
If no configuration is explicitly provided, the original configuration logic should remain unchanged, like "conf.Chunk.BlockSize * 8"

Reply:
removed